projects/js-packages/components/components/pricing-card/index.tsx

```diff
@@ -55,7 +55,8 @@ const PricingCard: React.FC< PricingCardProps > = ( {
 			) }
 			<h1 className="jp-components__pricing-card__title">{ props.title }</h1>
 			<div className="jp-components__pricing-card__pricing">
-				{ props.priceBefore !== props.priceAfter && props.priceAfter > 0 ? (
+				{ props.priceAfter === 0 && <LoadingPlaceholder width="100%" height={ 48 } /> }
```

Comment:
`priceAfter` is [0 by default](https://github.com/Automattic/jetpack/blob/c663031359d516cda5b72493f835a1440dcc150d/projects/packages/backup/src/js/components/Admin/no-backup-capabilities.js#L11). We should probably consider adding some kind of loading state instead of relying on `0`.